test/plugins/collections.test.js

```diff
@@ -267,6 +302,22 @@ describe('collection plugin test', () => {
             });
         });
 
+        it('returns a collection only with pipelines that exist', () => {
```

Comment:
This is to handle the case where a pipeline used to exist but got deleted. It should not cause an issue while a `GET` operation is performed.